app/src/test/kotlin/com/mapzen/erasermap/presenter/MainPresenterTest.kt

```diff
@@ -466,19 +468,88 @@ class MainPresenterTest {
     @Test fun onSearchResultSelected_shouldCenterOnCurrentFeature() {
         val result = Result()
         val features = ArrayList<Feature>()
+        features.add(Feature())
+        features.add(Feature())
+        val feature = SimpleFeature.create("1", "1", "", "",
+            "", "", "", "", "", "", "", 0.0, "", "", 40.0, 70.0)
+        features.add(feature.toFeature())
+        result.features = features
+        mainController.currentSearchItemPosition = 2
+        presenter.onSearchResultsAvailable(result)
+        presenter.onSearchResultSelected(0)
+        assertThat(mainController.currentSearchItemPosition).isEqualTo(2)
```

Comment:
This is a little confusing because there isn't a scenario in the app where the current search item position would be 2 but the search result selected would be 0...it would also be 2. I will update `presenter.onSearchResultSelected(0)` to select the second result. 

The current position doesn't get set to 0 because the index here is only used to determine which search pin should be active.